model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -42,23 +57,24 @@ def extract_holder_weights(constant_name, node_target, model, weights, to_numpy)
     Returns:
         Updated weights dictionary.
     """
-    named_parameters_weights = {constant_name: to_numpy(parameter) for name, parameter in
-                                model.named_parameters() if node_target == name}
-    named_buffer_weights = {constant_name: to_numpy(parameter) for name, parameter in
-                            model.named_buffers() if node_target == name}
+    named_parameters_weights = {
+        constant_name: to_numpy(parameter) for name, parameter in model.named_parameters() if node_target == name
+    }
+    named_buffer_weights = {
+        constant_name: to_numpy(parameter) for name, parameter in model.named_buffers() if node_target == name
+    }
     if len(named_parameters_weights) + len(named_buffer_weights) > 1:
         raise Exception(
-            f'Constant parameter can only have one tensor. Here we have '
-            f'{len(named_parameters_weights)+ len(named_buffer_weights)}')
+            f"Constant parameter can only have one tensor. Here we have "
+            f"{len(named_parameters_weights)+ len(named_buffer_weights)}"
+        )
 
     weights.update(named_parameters_weights)
     weights.update(named_buffer_weights)
     return weights
 
 
-def nodes_builder(model: GraphModule,
-                  module_dict: Dict,
-                  to_numpy: Callable) -> Tuple[List, List, List, Dict]:
+def nodes_builder(model: GraphModule, module_dict: Dict, to_numpy: Callable) -> Tuple[List, List, List, Dict]:
```

Comment:
Please remove this change

Reply:
damn formatting. I reverted the file to previous formating.